docs/node-support.md

```diff
@@ -10,7 +10,7 @@
 | 5.x              | 1.13.9 - 2.6.3               |
 | 6.x              | 2.9.0 - 3.9.x                |
 | 7.x              | 2.10.0 - 2.16.x              |
-| 8.x              | 2.13.3 - Current             |
+| 8.x              | 2.13.3 - 3.15.x             |
```

Comment:
I believe this will be 3.16, since beta will release with node 8 still turned on.

Reply:
I wasn't sure about that and meant to ask, thanks for noticing! I switched it to 3.16.x.